packages/native/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "@ledgerhq/native-ui",
   "version": "0.6.6",
-  "private": true,
```

Comment:
⚠️ This line is there to prevent accidental publishing, could you revert the change?

Reply:
(the publishing script takes care of formatting the package.json file: https://github.com/LedgerHQ/ui/blob/main/packages/native/scripts/prepareRelease.js#L25)